measurements/pages/__init__.py

```diff
@@ -275,6 +275,18 @@ def generator():
 
 @pages_blueprint.route('/files/download/<path:textname>')
 def files_download(textname):
+    if len(textname.split('/')) == 1:
```

Comment:
`'/' not in textname` is maybe a bit more idiomatic.

Reply:
OK